Library/Homebrew/test/test_cmd_audit.rb

```diff
@@ -13,53 +13,46 @@ def teardown
     FileUtils.rm_rf @dir
   end
 
-  def formula_text(name, text)
+  def formula_text(name, body = nil, patch = nil)
```

Comment:
Could you make an `options = {}` for the `patch`? Will make it a bit more readable.